### DISCUSSION TITLE
post/ebpf-talk-42-trampoline-on-x86/

### DISCUSSION THREAD

**Discussion by giscus** [Asphaltt/asphaltt.github.io#2] [Announcements]
# post/ebpf-talk-42-trampoline-on-x86/

类似 freplace，trampoline 是对 prologue 进行 poke 的另一个应用。 eBPF Talk: freplace on x86【汇编慎入】 TL;DR 类似 freplace，trampoline 是

https://asphaltt.github.io/post/ebpf-talk-42-trampoline-on-x86/

**Comment by lijianan98** [comment]
感谢分享！
